src/shared/conf-parser.c

```diff
@@ -1391,6 +1391,45 @@ int config_parse_permille(
         return 0;
 }
 
+int config_parse_io_cost_qos_scaling_percentage(
+                const char* unit,
```

Comment:
Formatting nit: the '*' should be attached to the argument name, not the type here.